services/app-api/utils/validation/schemaMap.test.ts

```diff
@@ -94,7 +94,6 @@ describe("Schemas", () => {
   ) => {
     for (let testCase of testCases) {
       let test = schemaToUse.isValidSync(testCase);
-
```

Comment:
Don't really need to add this space. Might as well have fewer changed files in this PR.